schemas/project.json

```diff
@@ -0,0 +1,67 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "resource:/project.json",
+  "title": "Project",
+  "description": "Schema to describe a project at lobid.",
+  "type": "object",
+  "properties": {
+    "id": {
+      "$ref": "resource:/id.json"
+    },
+    "type": {
+      "type": "array",
+      "items": {
+        "type": "string",
+        "const": "Project"
+      }
+    },
+    "alternateName": {
+      "$ref": "resource:/alternateName.json"
+    },
+    "description": {
+      "$ref": "resource:/description.json"
+    },
+    "name": {
+      "$ref": "resource:/name.json"
+    },
+    "image": {
+      "$ref": "resource:/image.json"
+    },
+    "starteDate": {
+      "$ref": "resource:/startDate.json"
+    },
+    "endeDate": {
+      "$ref": "resource:/startDate.json"
+    },
+    "funding": {
```

Comment:
I'd rather leave the funding out for now and remove it. We don't have anything like this in the example/instance data and no use case yet what we need here. This might come in the future but is not relevant now for the prototype.